tensortrade/environments/trading_environment.py

```diff
@@ -317,3 +320,7 @@ def close(self):
         for renderer in self._renderers:
             if callable(hasattr(renderer, 'close')):
                 renderer.close()  # pylint: disable=no-member
+
+    def _rewind(self):
+        if self._drop_n_frames > 0:
+            for i in range(self._drop_n_frames): self.feed.next()
```

Comment:
Do I need to call `self.clock.increment()` here too?

Reply:
No.